main.py

```diff
@@ -47,7 +58,7 @@ def get_release(user, repo_ver, name_re):
     return redirect(matched[0])
 
 def api_req(url):
-    resp = httpx.get(url)
+    resp = httpx.get(url, headers={'Authorization': config['GITHUB_AUTHORIZATION']}) if ast.literal_eval(config['GITHUB_IS_AUTHORIZED']) == True else httpx.get(url)
```

Comment:
I might want to simplify things a bit: if `GITHUB_TOKEN` is set and is non-empty, add the auth header.